packages/core/admin/admin/src/content-manager/history/components/VersionDetails.tsx

```diff
@@ -17,12 +20,35 @@ interface VersionHeaderProps {
 }
 
 const VersionHeader = ({ headerId, version }: VersionHeaderProps) => {
-  const { formatMessage } = useIntl();
+  const { formatMessage, formatDate, formatTime } = useIntl();
+  const { slug } = useParams<{
+    slug: UID.ContentType;
+  }>();
+  const { layout } = useContentTypeLayout(slug);
+
+  // The loading state is handled by the parent component
+  if (!layout) return null;
+
+  const mainFieldValue = version.data[layout.contentType.settings.mainField || 'id'] as string;
+  const formattedSubtitle = formatMessage(
+    {
+      id: 'content-manager.history.version.subtitle',
+      defaultMessage: '{hasLocale, select, true {{subtitle}, in {locale}} other {{subtitle}}}',
+    },
+    {
+      hasLocale: Boolean(version.locale),
+      subtitle: `${mainFieldValue} (${layout.contentType.info.singularName})`,
+      locale: version.locale?.name,
+    }
+  );
```

Comment:
ℹ️ For i18n here, using an injection zone would expose an area for other plugins to inject components into this header. Is that what we want just to display the i18n locale name here? 

For now I've handled it on the backend. Not trying to ignore your suggestion @joshuaellis, opened this pr to have the discussion, so let me know if you think this is an issue or if I've misunderstood something about injection zones / the hooks api.

@Marc-Roig also suggested an RFC or other doc to discuss i18n in v5. I think this will be a common problem if we leave i18n how it is now. I thought we had planned to make it a core plugin that could not be uninstalled? Anyway happy to write that document and help out anyway I can.

Reply:
I personally quite like what you have here 👍

Reply:
> I thought we had planned to make it a core plugin that could not be uninstalled

Even if it's a core plugin, it's domain driven development so content-manager shouldn't know anything about `locales`. I agree, its a common problem at the moment with the designs consistently assuming it's installed which is not correct.

> For i18n here, using an injection zone would expose an area for other plugins to inject components into this header. Is that what we want just to display the i18n locale name here?

I totally agree, it's overkill. If i'm being honest I would push back on design for this, it creates cross domain pollution. I think we need a meeting to discuss this because it's getting out of hand, we either remove i18n as a plugin and bring it into the CM or we continue with plugin approach, I don't mind personally. I think we need to stick to one path instead of doing a hybrid at the moment.

Reply:
I'd like to put a pin in the i18n stuff for now until we come to a unanimous direction, this sort of stuff impacts draft & publish too, but currently we're doing it in opposite ways which isn't good. What do you think or removing this i18n specific stuff for now?

Reply:
> I totally agree, it's overkill. If i'm being honest I would push back on design for this, it creates cross domain pollution. I think we need a meeting to discuss this because it's getting out of hand, we either remove i18n as a plugin and bring it into the CM or we continue with plugin approach, I don't mind personally. I think we need to stick to one path instead of doing a hybrid at the moment.

Definitely up for a meeting to discuss I can set something up for next week but it would be good to have Rémi around.

I think the locale display here is quite important for localized entries. Even if we push back on this topic I think we are likely to constantly be fighting product over this topic throughout the application and our technical constraints.

I'm leaning more towards leaving this addition for now with an action to refactor after we have a discussion in the coming weeks. 

I don't know exactly how the InjectionZone api works but as a workaround I was wondering if it would be possible to make like a private injection zone that only a set of white listed plugins could inject into... 🤷